project/src/const.ts

```diff
@@ -21,3 +21,9 @@ export const RatingTitles = [
   {value: '2', title: 'badly'},
   {value: '1', title: 'terribly'}
 ];
+
+export const URL_MARKER_DEFAULT =
+  '/img/pin.svg';
```

Comment:
Немного странное форматирование, длина строки ведь позволяет
```suggestion
export const URL_MARKER_DEFAULT = '/img/pin.svg';
```